client/actions/auth.js

```diff
@@ -178,8 +171,8 @@ const processTokens = (dispatch, apiToken, returnTo) => {
 
 export function loadCredentials() {
   return (dispatch) => {
-    const token = sessionStorage.getItem('delegated-admin:apiToken');
-    if (token || window.location.hash) {
```

Comment:
I can take care of this during a merge, but there are now two checks for window.location.hash.  I'm removing the outer check